tutorials/time-freq/plot_ssvep.py

```diff
@@ -0,0 +1,786 @@
+"""
+=======================================
+Basic analysis of an SSVEP/vSSR dataset
+=======================================
+
+In this tutorial we compute the frequency spectrum and quantify signal-to-noise
+ratio (SNR) at a target frequency in EEG data recorded during fast periodic
+visual stimulation (FPVS).
+Extracting SNR at stimulation frequency is a simple way to quantify frequency
+tagged responses in MEEG (a.k.a. steady state visually evoked potentials,
+SSVEP, or visual steady-state responses, vSSR in the visual domain,
+or auditory steady-state responses, ASSR in the auditory domain).
+
+DATA:
+
+We use a simple example dataset with frequency tagged visual stimulation:
+N=2 participants observed checkerboards patterns inverting with a constant
+frequency of either 12Hz of 15Hz. 10 trials of 20s length each.
```

Comment:
10 trials of 20s is probably a detail for `datasets_index.rst`

Reply:
agreed, the info should go there..

please give me a hand on which files i have to change. i saw the final [doc page](https://mne.tools/dev/overview/datasets_index.html#datasets) where it should go, but wasnt able to reverse engineer the source of the texts :)

also i didnt implement the dataset version correctly, yet.
so far it defaults to 0.7, according to `_dataset_version(path, name)` in `utils.py`
how do you usually handle this?

Reply:
> please give me a hand on which files i have to change.

[doc/overview/datasets_index.rst](https://github.com/mne-tools/mne-python/blob/main/doc/overview/datasets_index.rst)

> how do you usually handle this?

If you can add a `version.txt` in the actual archive for the data, that's best. But for now if you've already created them and don't want to upload a new version you can add a conditional here:

https://github.com/mne-tools/mne-python/blob/main/mne/datasets/utils.py#L162-L165

Reply:
alright thanks, i moved the info to `dataset_index.rst` and added a version.txt to the data repository